config/whitelabel.yml

```diff
@@ -90,6 +90,9 @@
   label_id: cologne
   default_locale: de
   recurring: third wednesday
+  custom_recurring:
+    :de: "jeweils am 3. Mittwoch in jedem 2. Monat (Januar, März, Mai, Juli, September, November) um 19:00 Uhr"
+    :en: "every 3rd Wednesday in every second month (January, March, May, July, September, November)"
```

Comment:
i think i would make this a flag and then put the text in a locale file `config/locales/XX.label.yml`. what do you think?